src/index.ts

```diff
@@ -33,14 +34,14 @@ function readFile$(file: string): Observable<string> {
  * Create the Angular Universal request handler
  * @param config 
  */
-export function angularUniversal({ index, main, staticDirectory, enableProdMode = false }: ServerConfiguration) {
+export function angularUniversal({ index, main, staticDirectory, enableProdMode = false, extraProviders }: ServerConfiguration) {
```

Comment:
What happens when `extraProviders` is undefined or null? Should we use a default of `[]`?

Reply:
Hi David, I think in the implementation of renderModuleFactory function, there is a check for null or undefined extraProviders. So we can leave it like this, or we can use [] as the default value.